### ISSUE TITLE
bug: component directive weird padding on top

### ISSUE THREAD

**Issue by ThorConzales** [driftyco/ionic#9890]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>2.0 RC4</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>6.0</span>  <span ionic-webview>webview</span>

<span ionic-description>After I upgraded to RC4 and when some element is injected as component directive then ~10px padding is added on top of the directive component that didn't happen before. This does not seem to happen on iOS. It is also reproducible with serve. I added a sample project and a screenshot where this mysterious character is seen that causes the padding.</span>

Project link: https://drive.google.com/open?id=0Byba--Lk0uJJUlR1MFNkaXYwb0k

![ionicpaddingbug](https://cloud.githubusercontent.com/assets/9930079/21683456/1afb786c-d362-11e6-9765-def64b700e99.png)

<span is-issue-template></span>

**Comment by jgw96** [comment]
Thanks for opening an issue with us! We will take a look at this.

**Comment by jgw96** [comment]
Hey @ThorConzales , unfortunately I cannot reproduce this issue. Would you mind posting a repo we can use to reproduce this issue? Thanks!

**Comment by ThorConzales** [comment]
Hi, since my project is proprietary enterprise grade app I cannot make it public. But are you saying that the sample project I attached does not yield the same result for you both while serving and while running on Android?

**Comment by ThorConzales** [comment]
This issue seems to be resolved in RC5.

**Comment by ThorConzales** [comment]
This issue is still reproducible in version 2.2.0 and it is now worse than before. Now it is happening on iOS too and before with the version 2.1.0 it didn't happen when you run the project with --prod key but now that does not seem to make any difference anymore. I attached 2 screenshots, first with the weird character being injected between the page content and component directive that causes the directive content to be pushed downwards and the second screenshot where I manually removed the character with chrome inspect and it did resolve the problem. It is still reproducible with the initial sample project I provided, just upgrade it to 2.2.0. I managed to reproduce it on 2 Android devices and 1 iOS device. 
![ionicbug1](https://cloud.githubusercontent.com/assets/9930079/23798458/71277d76-05ad-11e7-8e89-f8dad9891760.png)
![ionicbug2](https://cloud.githubusercontent.com/assets/9930079/23798465/7628aa2a-05ad-11e7-868c-79a2b1ff1b77.png)
